code/messaging/src/main/java/com/adobe/marketing/mobile/aepuitemplates/uimodels/AEPButton.kt

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepuitemplates.uimodels
+
+import com.adobe.marketing.mobile.aepuitemplates.utils.Constants
+
+/**
+ * Data class representing a button element in the UI.
+ *
+ * @property interactId The unique interaction ID for the button.
+ * @property actionUrl The URL to be opened when the button is clicked.
+ * @property text The text to be displayed on the button, represented by an [AEPText] object.
+ * @property borWidth The border width of the button.
+ * @property borColor The border color of the button.
+ *
+ * @param buttonSchemaMap A map containing key-value pairs to initialize the AEPButton properties.
+ */
+data class AEPButton(
```

Comment:
Let's keep naming consistent everywhere. We decided on `Aep` instead of `AEP` right?

Reply:
If we want to be consistent with iOS, we can keep it as `AEP`, but let's make sure we use the same format for all file names. @prudrabhat do you have a preference here?